install/helm/agones/values.yaml

```diff
@@ -280,6 +280,10 @@ gameservers:
   - default
   minPort: 7000
   maxPort: 8000
+  portRanges: []
```

Comment:
Chatting with @zmerlynn offline - we were discussing calling this `additionalPortRanges` - since it's in addition to `minPort` and `maxPort` ? WDYT?